### ISSUE TITLE
Implement storage description resource discovery

### ISSUE THREAD

**Issue by csarven** [nodeSolidServer/node-solid-server#1805]
https://solidproject.org/ED/protocol#server-storage-description details the discovery of the storage description resource.

>Servers MUST include the Link header field with rel="http://www.w3.org/ns/solid/terms#storageDescription" targeting the URI of the storage description resource in the response of HTTP GET, HEAD and OPTIONS requests targeting a resource in a storage.

---

https://github.com/solid/specification/issues/355#issuecomment-1140508784 shows how an application (e.g., https://dokie.li/ ) makes use of such resource to discover the features of the storage. Some common use cases are mentioned in https://github.com/solid/specification/issues/355#use-cases

**Comment by sumeetkolekarr** [comment]
Can I take this issue?

**Comment by bourgeoa** [comment]
@sumeetkolekarr branch `issue#1805` has been created

**Comment by sumeetkolekarr** [comment]
@bourgeoa Perfectly noted!

**Comment by csarven** [comment]
@aditya-hansraj Perhaps coordinate with @sumeetkolekarr in case they're already working on it or have implemented? What needs to be implemented is mentioned in the first comment. You can also see the Matrix chat for NSS ( https://matrix.to/#/#solid_node-solid-server:gitter.im ) if you'd like to coordinate with the others.

**Comment by aditya-hansraj** [comment]
@sumeetkolekarr hello, are you still working on this issue ?

**Comment by sumeetkolekarr** [comment]
@aditya-hansraj I am still working on it, if you want, we both can work on the issue

**Comment by aditya-hansraj** [comment]
@sumeetkolekarr yes definitely, you can just fill me up on whatever you've done till now, then we can solve this issue together.
